pilot/pkg/credentials/kube/secrets.go

```diff
@@ -53,6 +53,8 @@ const (
 	TLSSecretCert = "tls.crt"
 	// The ID/name for the k8sKey in kubernetes tls secret.
 	TLSSecretKey = "tls.key"
+	// The ID/name for the certificate OCSP staple in kubernetes tls secret
+	TLSSecretOcspStaple = "tls.staple"
```

Comment:
```suggestion
	TLSSecretOcspStaple = "tls.ocsp-staple"
```

Minor thing, but since "ocsp staple" is used everywhere else, shouldn't we use that (or something like it) here? `staple` is a purely OCSP thing and not a TLS primitive.

Reply:
Fixed.